### DISCUSSION TITLE
ScreenToGif 2.36

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1076] [Announcements]
# What's new?

• New installer and portable versions with the full package (no .NET 6 download required) are available alongside with the lighter versions, which still require the installation of .NET 6 desktop runtime.  
• New installer package (MSIX) available.  
• Added an option to prompt to overwrite when saving (enabled by default).  
• Updated the Danish, French, German, Hungarian, Polish, Norwegian, Russian, and Simplified Chinese localizations.  

### Fixed:

♦ The smooth loop feature was not working properly.  
♦ A new message will be displayed if you already have a smooth loop based on current settings instead of a warning.  
♦ It was not possible to set the app to start at Windows startup.  
♦ A settings migration issue from 2.31 to newer versions was fixed.  
♦ It was not possible to export as PSD.  
♦ When not having permission to save the settings to the installation location, the app would not try to save to AppData.  

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.36'>ScreenToGif 2.36</a>.</em>

**Comment by anatimita** [comment]
Hi!
Please, i need some help. I started having a problem with this wonderful and helpful tool :(
I can't figure out why a 235 KB gif (for example), after deleting some frames, it's saved way too large (2.82 MB) than the initial encoded gif. 
I even tried by installing an older version but the problem persists.
I must mention that I had to delete some files from AppData/ temp/ Recording because it no longer allowed me to save gifs due to lack of space.

Thank you,
Ana

**Reply by NickeManarin** [reply]
Which encoder (and encoder settings) are you using? 
If the gif originated from other tool, the encoding parameters would not match and the gif size would be different.

**Reply by anatimita** [reply]
I only use ScreenToGif. The original gif was recorded with ScreenToGif and the editor is the same.
I've attached a screenshot of the encoder settings (which I've never changed)

![image](https://user-images.githubusercontent.com/103458031/162919385-7386bd3a-3ccd-4e98-8939-ed5c536ce06e.png)

**Reply by anatimita** [reply]
I changed the "Default" preset to "High quality" and it works.

![image](https://user-images.githubusercontent.com/103458031/162924843-49729cd3-3c24-49a4-ab2a-81a77d5bf7fc.png)

**Reply by NickeManarin** [reply]
In the first screenshot, the option "Detect unchanged pixels" was unchecked.
That option really plays out in reducing the file size.

You can click on the reset button bellow the preset in order to reset its settings.

**Comment by anatimita** [comment]
Nice! 
Thank you for your time and tips.
You're doing a great job, keep going.

Best regards.

**Comment by RED-CAD-Solutions-AG-Development** [comment]
Hello 

Since the update to 2.37, the GiF files are getting extremely large. 

I have not made any changes to my settings.

This is strange.

Best regards

**Reply by NickeManarin** [reply]
Hi, can you show me which encoder (and its settings) are you using to export?
If using the built in encoder (ScreenToGif) then remember to check the option to detect unchanged pixels (and the option to use a chroma key).

**Reply by RED-CAD-Solutions-AG-Development** [reply]
Hello 

Thank you for your reply. 

Yes of course :-)

![grafik](https://user-images.githubusercontent.com/51949837/175037474-de1e979c-043b-4bad-a60b-3bb3e2c1c206.png)

I tested all four presets with different color settings 
![grafik](https://user-images.githubusercontent.com/51949837/175037879-f0e8d699-139a-4b17-aa3b-77868f281734.png)

![grafik](https://user-images.githubusercontent.com/51949837/175038205-5b491ffa-e4f0-4ef9-b096-ea55ebc917e9.png)

Is maybe a 4K Monitor a problem?
